stored_requests/backends/file_fetcher/fetcher.go

```diff
@@ -35,8 +36,15 @@ func (fetcher *eagerFetcher) FetchRequests(ctx context.Context, requestIDs []str
 	return storedRequests, storedImpressions, errs
 }
 
+// Fetch Responses - Implements the interface to read the stored response information from the fetcher's FileSystem, the directory name is "stored_responses"
 func (fetcher *eagerFetcher) FetchResponses(ctx context.Context, ids []string) (data map[string]json.RawMessage, errs []error) {
```

Comment:
Nitpick: what do you think about renaming `fetcher` here to just be `f`

Reply:
@AlexBVolcy i think that would be a different discussion, all the other fetchers, e.g `http_fetcher`, `empty_fetcher and  `db_fetcher` are all using the same `fetcher` as the receiver name